soc/xtensa/intel_adsp/ace_v1x/power.c

```diff
@@ -49,38 +49,24 @@ __weak void pm_state_set(enum pm_state state, uint8_t substate_id)
 		} else {
 			k_cpu_idle();
 		}
-
-		break;
-	case PM_STATE_SUSPEND_TO_IDLE: /* D0ix */
-		__fallthrough;
-	case PM_STATE_RUNTIME_IDLE:/* D0 */
```

Comment:
PM_STATE_RUNTIME_IDLE it is not needed and if @mwasko doesn't mind it can also be removed from the documentation. From what I understand, on MTL it should behave the same as PM_STATE_ACTIVE.

PM_STATE_SUSPEND_TO_IDLE on the other hand needs to be implemented and this is something that I am currently working on. I do not know if it makes sense to remove it now, since it will be added back in a moment.

Reply:
The way it is now is confuse. I rather have something consistent and clean in upstream. We can always change back when needed.